routing.rst

```diff
@@ -27,7 +27,20 @@ route that can match *any* URL like ``/blog/my-post`` or
 ``/blog/all-about-symfony``.
 
 Routes can be configured in YAML, XML, PHP or annotations. All formats provide
-the same features and performance, so choose the one you prefer:
+the same features and performance, so choose the one you prefer. If you choose
+PHP annotations, run this command once in your app to add support for them:
```

Comment:
We recently started replacing all "app" by "application".

Reply:
Fixed.